f/connectors/alerts/tests/alerts_gcs_test.py

```diff
@@ -85,3 +95,6 @@ def test_script_e2e(pg_database, mock_alerts_storage_client, tmp_path):
         assert (
             asset_storage / "100/2023/09/202309900112345671/images" / f"{basename}.jpg"
         ).exists()
+
+    # Alerts metadata is not saved to disk
+    assert not (asset_storage / "alerts_history.csv").exists()
```

Comment:
Arguably, this assertion can be omitted, since there is no reason to expect this moving forward. But I thought to keep it to ensure that we don't accidentally save this CSV in the future, since it has already happened.